SDSetupBackend/Controllers/v1.cs

```diff
@@ -86,7 +86,7 @@ public class v1 : ControllerBase {
                     DeletingFileStream stream = (DeletingFileStream) ZipFromFilestreams(files.ToArray(), uuid);
 
                     Program.generatedZips[uuid] = stream;
-                    stream.Timeout(30000);
+                    stream.Timeout(180000);
```

Comment:
You included the changes from your other PR here

Reply:
I thought I rolled back/extracted this change from this PR.  I'll fix that.